src/main/io/gps.h

```diff
@@ -68,6 +214,14 @@ typedef enum {
     UBLOX_MODEL_AIRBORNE_4G,
 } ubloxModel_e;
 
+typedef enum {
+    UBLOX_UTC_STANDARD_AUTO = 0,
+    UBLOX_UTC_STANDARD_USNO = 3,
+    UBLOX_UTC_STANDARD_EU = 5,
+    UBLOX_UTC_STANDARD_SU = 6,
+    UBLOX_UTC_STANDARD_NTSC = 7,
+} ubloxUtcStandard_e;
```

Comment:
Wouldn't `UBLOX_UTC_STANDARD_USNO` then end up being 1 rather than 3? Same for the others where there are gaps between the numeric values, i.e. things aren't just incrementing from 0. Or are the numeric values unimportant?

Reply:
The UTC standards have to be specific values when used in the configuration messages. If we change that enum like that, the values will change. The values need to be:
```
typedef enum {
    UBLOX_UTC_STANDARD_AUTO = 0,
    UBLOX_UTC_STANDARD_USNO = 3,
    UBLOX_UTC_STANDARD_EU = 5,
    UBLOX_UTC_STANDARD_SU = 6,
    UBLOX_UTC_STANDARD_NTSC = 7,
} ubloxUtcStandard_e;
```
Unless we do some other mapping to send those values in the `utcStandard` field of UBX-CFG-NAV5 or the `CFG-NAVSPG-UTCSTANDARD` key/value of the UBX-CFG-VALSET message...

Reply:
Alright, I was not aware of the definition in the docs. Then I'll look into it again.